### ISSUE TITLE
ERROR in ./~/openpgp/dist/openpgp.min.mjs Module build failed: SyntaxError: "r" is read-only

### ISSUE THREAD

**Issue by muhammadali298** [openpgpjs/openpgpjs#1301]
I have updated the version from `3.0.8` to `5.0.0-2` and I am getting following error.

```
ERROR in ./~/openpgp/dist/openpgp.min.mjs
Module build failed: SyntaxError: "r" is read-only
```

I am not sure what should I change in in my code. I assume the issue is with the loader in my webpack. Here is a sample of my webpack configurations

```
module: {
    noParse: /node_modules\/quill\/dist/,
    rules: [
      { test: /\.mjs$/, use: 'babel-loader', include: /node_modules\/openpgp/ }
    ]
  }
```

**Comment by jpreynat** [comment]
I have the same issue with `5.0.0-2` but with this error from esbuild:
```
node_modules/openpgp/dist/node/openpgp.min.js:2:230359: error: Cannot assign to "r" because it is a constant
```

It took me a while to understand the reason, but it seems that the fork `@openpgp/pako` that you've been using for building this version of `openpgp` has an invalid `const` declaration instead of a `let` for `dictLength` here (I highlighted the code from the declaration to the reassignment to the `const`):
https://github.com/openpgpjs/pako/blob/f38f7368a5fa511e54b95add2f04444c3a9d803f/lib/zlib/deflate.js#L1744-L1787

I've seen that you've updated the dependencies and build phase since then.
It would be great if you could release a new version as the current one cannot build for Node.

**Comment by twiss** [comment]
This should be fixed by 5ae7fc4444a384959e2f010cd5b821fc1a357823 in [v5.0.0-3](https://github.com/openpgpjs/openpgpjs/releases/tag/v5.0.0-3).

**Comment by jpreynat** [comment]
Thanks @twiss, this has indeed fixed the issue 👍
